js/pulse.js

```diff
@@ -359,10 +359,14 @@
 
                 if(!this._pulseHost) {
                     log('No Pulse hostname found in plugin parameters or media metadata; will not attempt to show Pulse ads');
+                    this.ready = true;
+                    amc.onAdManagerReady();
                     return;
                 }
 
                 this.ready = true;
+                amc.onAdManagerReady();
```

Comment:
Initial plays can be triggered any time after onAdManagerReady. This API will notify the AMC that the ad plugin is ready. Once all ad plugins are ready, the AMC will unblock the initial play, so the user can start playback by clicking on the video.